ociinstaller/versionfile/versionfile.go

```diff
@@ -68,16 +69,29 @@ func read(path string) (*VersionFile, error) {
 	var data VersionFile
 
 	if err := json.Unmarshal([]byte(file), &data); err != nil {
-		fmt.Println("---- error: ", err)
+		log.Println("Error while reading version file", err)
 		return nil, err
 	}
 
+	if data.Plugins == nil {
+		// create an empty Map with room for at least 2 plugins
```

Comment:
`data.Plugins == nil` becomes true when no plugins are installed.

putting in `2` so that we have room during the first installation, without having to resize

Reply:
is resizing cost really an issue here - not sure I agree on choosing an abitrary initial size - we'll have to resize for the third right?